db/schema.rb

```diff
@@ -1216,31 +1216,43 @@
     t.index ["deleted_at"], name: "index_transport_orders_on_deleted_at", using: :btree
   end
 
+  create_table "transport_requisition_details", force: :cascade do |t|
```

Comment:
@nathnael @dawitd what's the difference between tranport_requisition_items and transport_requisition_details?